browser/src/layer/tile/ContentControlSection.ts

```diff
@@ -68,6 +68,12 @@ export class ContentControlSection extends CanvasSectionObject {
 
 		if (json.date) {
 			this.sectionProperties.datePicker = true;
+			var languageAndCode = this.map['stateChangeHandler'].getItemValue('.uno:LanguageStatus');
```

Comment:
It's the document language not the UI language, right?
I can accept this, if desktop LibreOffice and/or MS Word work the same.